content/dev-tools/utils/websocketapi-modals.tsx

```diff
@@ -0,0 +1,282 @@
+import * as React from "react";
```

Comment:
Odd that importing React is required.  The need for this was eliminated a while ago.  The explorer now doesnt require it.